tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1337,6 +1304,9 @@ void ROOT::Experimental::Internal::RNTupleFileWriter::Commit()
    if (fIsBare) {
       RTFNTuple ntupleOnDisk(fNTupleAnchor);
       fFileSimple.Write(&ntupleOnDisk, ntupleOnDisk.GetSize(), fFileSimple.fControlBlock->fSeekNTuple);
+      // Append the checksum
+      std::uint64_t checksum = XXH3_64bits(ntupleOnDisk.GetPtrCkData(), ntupleOnDisk.GetSizeCkData());
```

Comment:
It looks like the checksum in this case and the `::Streamer` case is calculated over the same range of bytes but I can not tell for sure (at least not from the PR).
So related questions (which solves the above question) is do we have a test where we write with the RMiniFIle and read with TFile and vice et versa.  If we don't can we please add one?

Reply:
In `ntuple_basics.cxx` there is a `FileAnchor` test which creates a RNTuple and writes it via RMiniFile, then it updates it and writes it back through TFile, and then it opens again through TFile.
There is also in the same file the `MultipleInFile` test which writes a RNTuple through TFile and then reads it through RMiniFile.
In fact, thanks to those tests I found some inconsistencies I had introduced between RMiniFile and TFile while making this change.